extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfoItemExtractor.java

```diff
@@ -127,4 +127,14 @@ public interface StreamInfoItemExtractor extends InfoItemExtractor {
     default String getShortDescription() throws ParsingException {
         return null;
     }
+
+    /**
+     * Check if the stream is a short-form content
+     *
+     * @return {@code true} if the stream is a short-form content
+     * @throws ParsingException thrown if there is an error in the extraction
+     */
```

Comment:
You should use the first description everywhere (with my changes, I think), which give us more details about what is this type of contents.